src/main/scala/mesosphere/marathon/core/storage/store/impl/zk/ZkPersistenceStore.scala

```diff
@@ -31,12 +31,13 @@ case class ZkId(category: String, id: String, version: Option[OffsetDateTime]) {
 
   // BUG: id = "" for the root group this results in "Path must not end with / character" in curator
   def path: String = version.fold(f"/$category/$bucket%x/$id") { v =>
-    f"/$category/$bucket%x/$id/${ZkId.DateFormat.format(v)}"
+    f"/$category/$bucket%x/$id/${ZkId.WriteDateFormat.format(v)}"
   }
 }
 
 object ZkId {
-  val DateFormat = DateTimeFormatter.ISO_OFFSET_DATE_TIME
+  val WriteDateFormat = Timestamp.WriteFormatter // WriteDateFormat is following our formatting style
```

Comment:
What would you think if we used the naming Timestamp.StrictISO8601 and Timestamp.FlexibleISO8601 ?

Reply:
We'd have to rename the vals if we ever change it so it seems like an implementation detail to me.